### PR TITLE
Update henry.py

### PR DESCRIPTION
**Pull request by wanda-carlson**
Withbody

### CODE REVIEW

File: src/henry.py [hunk 1]
```diff
@@ -1 +1,2 @@
 print("Henry testing v0.8.0")
+print("Henry testing v0.8.0")
```
Comment (wanda-carlson) [inline src/henry.py:2]:
```suggestion
print("Wanda testing v0.8.0")
```

File: src/henry.py [hunk 2]
```diff
@@ -1 +1,2 @@
 print("Henry testing v0.8.0")
```
Comment (wanda-carlson) [inline src/henry.py:1]: Two

**Comment by wanda-carlson** [comment]
@henry-richard

**Comment by wanda-carlson** [comment]
@henry-richard check

File: src/henry.py [hunk 3]
```diff
@@ -1 +1,2 @@
...
+print("Wanda testing v0.8.0")
```
Comment (henry-richard) [inline src/henry.py:2]:
```suggestion
print("Wanda testing v0.8.0, yes")
```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request merged**